MyPooDiary/MyPooDiary/Screens/Report/ReportViewController.swift

```diff
@@ -110,88 +92,65 @@ extension ReportViewController {
-            )
-        }
+        reportTableView.register(ReportTableViewCell.self,
+                                 forCellReuseIdentifier: ReportTableViewCell.identifier
+        )
+        reportTableView.register(TotalReportCell.self, forCellReuseIdentifier: TotalReportCell.identifier)
+    }
     
 }
 
 extension ReportViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         switch section {
         case 0 :
+            return 1
+        case 1 :
             return 2
-        case 1:
-            return 3
         case 2:
+            return 3
+        case 3:
             return 5
         default:
             return 0
         }
     }
     
-    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        return headerContainerView
-        
-        
-    }
     func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
         return sectionName[section]
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        guard let reportCell = tableView.dequeueReusableCell(
-            withIdentifier: reportTableViewCell.identifier, for: indexPath)
-                as? reportTableViewCell else { return UITableViewCell() }
-        print(indexPath.section)
+        
         switch indexPath.section {
         case 0 :
-            reportCell.dataBind(model: dummy1[indexPath.row])
+            let cell = tableView.dequeueReusableCell(withIdentifier: TotalReportCell.identifier, for: indexPath) as! TotalReportCell
+            cell.dataBind(model: dummy0[indexPath.row])
+            return cell
         case 1:
-            reportCell.dataBind(model: dummy2[indexPath.row])
+            let cell = tableView.dequeueReusableCell(withIdentifier: ReportTableViewCell.identifier, for: indexPath) as! ReportTableViewCell
+            cell.dataBind(model: dummy1[indexPath.row])
+            return cell
         case 2:
-            reportCell.dataBind(model: dummy3[indexPath.row])
+            let cell = tableView.dequeueReusableCell(withIdentifier: ReportTableViewCell.identifier, for: indexPath) as! ReportTableViewCell
+            cell.dataBind(model: dummy2[indexPath.row])
+            return cell
+        case 3:
+            let cell = tableView.dequeueReusableCell(withIdentifier: ReportTableViewCell.identifier, for: indexPath) as! ReportTableViewCell
+            cell.dataBind(model: dummy3[indexPath.row])
+            return cell
         default:
-            reportCell.dataBind(model: dummy1[indexPath.row])
+            let cell = tableView.dequeueReusableCell(withIdentifier: ReportTableViewCell.identifier, for: indexPath) as! ReportTableViewCell
+            cell.dataBind(model: dummy1[indexPath.row])
+            return cell
         }
```

Comment:
여기는 같은 cell에서 있는 함수 호출하니까 이렇게 해도 될거같아요
그리고 더미1, 2, 3보다는 이런명명이 더 직관적이고 좋을거같아여

```suggestion
    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        guard let reportCell = tableView.dequeueReusableCell(
            withIdentifier: reportTableViewCell.identifier, for: indexPath)
                as? reportTableViewCell else { return UITableViewCell() }
        switch indexPath.section {
        case 0 :
            reportCell.dataBind(model: like[indexPath.row])
        case 1:
            reportCell.dataBind(model: strength[indexPath.row])
        case 2:
            reportCell.dataBind(model: color[indexPath.row])
        default:
            break
        }
        return reportCell
```

Reply:
<img width="1441" alt="image" src="https://user-images.githubusercontent.com/80062632/204965983-b6d36c6e-c135-4fe9-b54b-07ea69258395.png">

Reply:
수정했슴니당 🔆